### PR TITLE
NAS-114661 / None / NAS-114661: Refactoring boot stats/settings dialog

### PR DESCRIPTION
**Pull request by undsoft**
Testing: Settings -> Boot -> Actions -> Stats/Settings.
Date is now shown in server timezone.

### CODE REVIEW

File: src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<h1 matDialogTitle>{{ 'Stats/Settings' | translate }}</h1>
+<form
+  *ngIf="state"
+  [formGroup]="form"
+  class="ix-form-container"
+  (submit)="onSubmit()"
+>
+  <div class="status-item">
+    <span class="status-name">
+      {{ 'Boot Pool Condition:' }}
```
Comment (AlexKarpov98) [inline src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.html:10]: {{ 'Boot Pool Condition:' | translate  }}

File: src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    <span class="status-value">
+      {{ condition | mapValue: poolStatusLabels }}
+      <ng-container *ngIf="condition === PoolStatus.Degraded">
+        {{ 'Check Alerts for more details.' | translate }}
+      </ng-container>
+    </span>
+  </div>
+  <div class="status-item">
+    <span class="status-name">
+      {{ 'Size:' }}
```
Comment (AlexKarpov98) [inline src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.html:21]: {{ 'Size:' | translate }}

File: src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    </span>
+    <span class="status-value">
+      {{ state?.properties?.size?.parsed | filesize: { standard: 'iec' } }}
+    </span>
+  </div>
+  <div class="status-item">
+    <span class="status-name">
+      {{ 'Used:' }}
```
Comment (AlexKarpov98) [inline src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.html:29]: {{ 'Used:' | translate }}

File: src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.ts [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+@UntilDestroy()
+@Component({
+  templateUrl: './bootenv-stats-dialog.component.html',
+  styleUrls: ['./bootenv-stats-dialog.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class BootenvStatsDialogComponent implements OnInit {
+  form = this.fb.group({
+    interval: [null as number, [Validators.required, Validators.min(1)]],
```
Comment (AlexKarpov98) [inline src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.ts:26]: `interval: [null as number, [Validators.required, numberValidator(), Validators.min(1)]],`

**Review by AlexKarpov98 — CHANGES_REQUESTED** [review]
Look good!
Screenshot_5

File: src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.ts (hunk 4 above)
Comment (undsoft) [reply src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.ts:26]: Fixed the translations, but I don't see why I need to have numberValidator(). We've agreed that `type` in ix-input should provide validation for number.

Comment (AlexKarpov98) [reply src/app/pages/system/bootenv/bootenv-stats-dialog/bootenv-stats-dialog.component.ts:26]: Sorry, You are right. `ix-input` has own validator for type.

**Review by AlexKarpov98 — APPROVED** [review]

**Pull request merged**